Patches/Vanilla Apparel Expanded/ThingDefs_Misc/Apparel_Industrial.xml

```diff
@@ -1,121 +1,176 @@
+				</li>
 
-			<!-- == VAE_Apparel_LabCoat == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/statBases/ArmorRating_Sharp</xpath>
-				<value>
-					<Bulk>7.5</Bulk>
-					<WornBulk>1</WornBulk>
-					<ArmorRating_Sharp>0.1</ArmorRating_Sharp>
-				</value>
-			</li>
+				<!-- == VAE_Apparel_LabCoat == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<Bulk>7.5</Bulk>
+						<WornBulk>1</WornBulk>
+						<ArmorRating_Sharp>0.1</ArmorRating_Sharp>
+					</value>
+				</li>
+				<!-- cloth cost -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/costList/Cloth</xpath>
+					<value>
+						<Cloth>40</Cloth>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_SheriffShirt == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_SheriffShirt"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<StuffEffectMultiplierArmor>1.1</StuffEffectMultiplierArmor>
-				</value>
-			</li>
-		</operations>
+				<!-- == VAE_Apparel_SheriffShirt == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_SheriffShirt"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<StuffEffectMultiplierArmor>1.1</StuffEffectMultiplierArmor>
+					</value>
+				</li>
+
+				<!-- == costStuffCounts == -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_TankTop"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>15</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_CasualTShirt" or defName="VAE_Apparel_Shorts" or defName="VAE_Apparel_Skirt" or defName="VAE_Apparel_Trousers"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>20</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_ShirtandTie" or defName="VAE_Apparel_Overalls" or defName="VAE_Apparel_FleeceShirt" or defName="VAE_Apparel_SheriffShirt"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>25</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_Hoodie" or defName="VAE_Apparel_Jeans" or defName="VAE_Apparel_ChefsUniform" or defName="VAE_Apparel_MilitaryUniform"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>30</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_SuitJacket" or defName="VAE_Apparel_MilitaryJacket" or defName="VAE_Apparel_Jumpsuit"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>40</costStuffCount>
```

Comment:
I think the jumpsuit and the military uniform above should be turned into their own tier, since they're "larger" outfits with far more coverage, long pants and sleeves and an undershirt, and made of fairly sturdy material.

So, I think 50 - 80 material would be appropriate.